sormas-ui/src/main/java/de/symeda/sormas/ui/caze/AbstractCaseView.java

```diff
@@ -251,9 +254,13 @@ public ViewMode getViewMode() {
 
 	public void setCaseEditPermission(Component component) {
 
-		Boolean isCaseEditAllowed = FacadeProvider.getCaseFacade().isCaseEditAllowed(getReference().getUuid());
+		Boolean isCaseEditAllowed = isCaseEditAllowed();
```

Comment:
It was the same before my changes, I think the reason for the if is that if `component.setEnabled` is called with true it will enable all the fields also the ones should always be disabled.